src/static/riot/competitions/detail/participant_manager.tag

```diff
@@ -0,0 +1,108 @@
+<participant-manager>
+    <div show="{participants}">
+        <div class="ui icon input">
+            <input type="text" placeholder="Search..." ref="participant_search" onkeyup="{ search_participants }">
+            <i class="search icon"></i>
+        </div>
+        <select ref="participant_status" class="ui dropdown" onchange="{ update_participants.bind(this, undefined) }">
+            <option value="">Status</option>
+            <option value="-">----</option>
+            <option value="approved">Approved</option>
+            <option value="pending">Pending</option>
+            <option value="denied">Denied</option>
+            <option value="unknown">Unknown</option>
+        </select>
+        <table class="ui celled striped table">
+            <thead>
+            <tr>
+                <th>Username</th>
+                <th>Email</th>
+                <th>Status</th>
+                <th class="center aligned">Actions</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr each="{participants}">
+                <td>{username}</td>
+                <td>{email}</td>
+                <td>{_.startCase(status)}</td>
+                <td class="right aligned">
+                    <button class="mini ui red button icon" show="{status !== 'denied'}"
+                            onclick="{ revoke_permission.bind(this, id) }"
+                            data-tooltip="Revoke"
+                            data-inverted=""
+                            data-position="bottom center">
+                        <i class="close icon"></i>
+                    </button>
+                    <button class="mini ui green button icon" show="{status !== 'approved'}"
+                            onclick="{ approve_permission.bind(this, id) }"
+                            data-tooltip="Approve"
+                            data-inverted=""
+                            data-position="bottom center">
+                            <i class="checkmark icon"></i>
+                    </button>
+                    <button class="mini ui blue button icon"
+                            data-tooltip="Send Message"
+                            data-inverted=""
+                            data-position="bottom center"
+                            onclick="">
```

Comment:
pull this onclick? -- and other empty attributes around here?

Reply:
pointed the `onlick` to a placeholder function for dealing w/ sending emails. (logic that doesn't exist yet'
the `data-inverted=''` has to remain there so that the tooltip will be dark instead of light. It's a weird Semantic thing